scripts/run_example

```diff
@@ -94,12 +95,19 @@ else
   readonly TLS_ARGS=("--ca_cert=${SCRIPTS_DIR}/../examples/certs/local/ca.pem")
 fi
 
+readonly CLIENT_ARGS=("${@}")  # Choose client args provided after '--'.
+
 # Run the application client.
-if [ $# -eq 0 ]; then
-  "./bazel-client-bin/examples/${EXAMPLE}/client/client" "${TLS_ARGS[@]}" "${ADDITIONAL_ARGS[@]-}"
-else
-  readonly CLIENT_ARGS=("${@}")  # Choose client args provided after '--'.
-  "./bazel-client-bin/examples/${EXAMPLE}/client/client" "${TLS_ARGS[@]}" "${ADDITIONAL_ARGS[@]-}" "${CLIENT_ARGS[@]-}"
-fi
+case "${client}" in
+  rust)
+    # Rust example parameters only support `-` as a delimeter, and C++ ones only `_`.
+    cargo run --release --target=x86_64-unknown-linux-musl --manifest-path="${SCRIPTS_DIR}/../examples/${EXAMPLE}/client/rust/Cargo.toml" -- \
```

Comment:
Some flags quote the flag names, some only the value, please make them consistent if possible.

Reply:
Done